### ISSUE TITLE
CHECK THE PHONE NUMBER FOR "IMPORTABILITY" does not work with Java Iris SDK

### ISSUE THREAD

**Issue by feanor07** [Bandwidth/java-bandwidth-iris#25]
When I try to check the phone number for importability as advised in the documentation at https://dev.bandwidth.com/numbers/guides/hostedMessaging.html#check-for-importability I always get the following exception:

```
com.bandwidth.iris.sdk.IrisClientException: <?xml version="1.0" encoding="UTF-8" standalone="yes"?><ImportTnCheckerResponse><ResponseStatus><ErrorCode>5070</ErrorCode><Description>Telephone number is invalid.</Description></ResponseStatus></ImportTnCheckerResponse>
	at com.bandwidth.iris.sdk.IrisClient.executeRequest(IrisClient.java:178)
	at com.bandwidth.iris.sdk.IrisClient.post(IrisClient.java:100)
	at com.bandwidth.iris.sdk.IrisClient.post(IrisClient.java:91)
	at com.bandwidth.iris.sdk.model.ImportTnChecker.Check(ImportTnChecker.java:9)
....
```

After debugging I found that this is the xml payload generated via using `ImportTnChecker`'s `Check` method as in the documentation:

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ImportTnCheckerPayload>
    <TelephoneNumbers>
        <TelephoneNumber>
            <FullNumber>5554443333</FullNumber>
        </TelephoneNumber>
    </TelephoneNumbers>
</ImportTnCheckerPayload>
```

This does not match with the example provided in the HTTP part of the documentation; the correct XML should have been:

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ImportTnCheckerPayload>
    <TelephoneNumbers>
        <TelephoneNumber>5554443333</TelephoneNumber>
    </TelephoneNumbers>
</ImportTnCheckerPayload>
```
which I tested with postman and it works.

TLDR; when we use `ImportTnChecker` as described in the documentation it simply fails due to incorrect xml marshalling!

**Comment by jchavez443** [comment]
We have resolved this issue in version `1.12` thank you for the catch!
